source/_integrations/systemmonitor.markdown

```diff
@@ -31,6 +31,12 @@ All sensors are also marked as diagnostic and won't be automatically added to au
 
 {% endnote %}
 
+### Enable entity
+1. Navigate to System Monitor integration.
+2. Select entity.
+3. Click on gear icon.
+4. Toggle "Enable" to ON/OFF.
```

Comment:
```suggestion

To enable or disable an entity, follow the steps on [enabling or disabling an entity](https://www.home-assistant.io/common-tasks/general/#enabling-or-disabling-entities).
```
I'm not sure if you have seen it, but we have a procedure on this in the Common tasks section of the docs. I think it is better to refer to those instead of duplicating the steps here.